packages/wasm/package.json

```diff
@@ -5,8 +5,6 @@
   "license": "MIT",
   "type": "module",
   "scripts": {
-    "download-keys": "tsx ./utils/download-proving-keys.ts",
-    "precompile": "npm run download-keys",
```

Comment:
I rather liked the use of a `precompile` script. I'm partial to using more of that in the codebase

`download-keys` is also used in the context of the ci pipeline since it was missing keys: https://github.com/penumbra-zone/web/blob/config-pk/.github/workflows/turbo-ci.yml#L116